external/nyc311/tests.py

```diff
@@ -29,8 +30,8 @@ def test_fetch_311_data(self, MockSocrata):  # noqa: N803
         client.get.assert_called_once()
 
 
-@mock.patch("external.nyc311.fetch.fetch_311_data", fetch_311_data_stub)
 class NYC311StatiscticsTests(TestCase):
+    @mock.patch("external.nyc311.fetch.fetch_311_data", fetch_311_data_stub)
```

Comment:
how does this work? when I was working on the geocoder story I could never get it to actually call the stub when i structured the `@mock.patch` annotation this way. It would always call the real API.

Reply:
I guess the key difference is that I'm using `get_311_statistics` and patching `fetch_311_data`.

Reply:
oh haha makes sense